srcs/incs/CommonDefinitions.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef SRCS_INCS_COMMONDEFINITIONS_HPP_
+#define SRCS_INCS_COMMONDEFINITIONS_HPP_
+
+#include <arpa/inet.h>
```

Comment:
```suggestion
#include <stdint.h>
```

Reply:
I was trying to add #include <cstdint>
and it was complaining that it needed c++17 (11?), and I decided to keep that one.
I didn't notice that the problem was that I was NOT including the native c header, but rather the wrapper.
Thx!